satellite/orders/orders_test.go

```diff
@@ -131,6 +131,8 @@ func TestUploadDownloadBandwidth(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, expectedData, data)
 
+		// We need enough time to pass after the download ends for storagenodes to save orders
+		time.Sleep(200 * time.Millisecond)
```

Comment:
note for reviewers - the `Download()` call finishing faster means that there is not always enough time for storagenodes to save orders before we calculate totals in this test. I do not know how to sync this test without a time.Sleep

Reply:
Yeah, I also don't see an alternative way.